csharp/extractor/Semmle.Extraction.CSharp/Entities/Expressions/ImplicitToString.cs

```diff
@@ -0,0 +1,59 @@
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Semmle.Extraction.CSharp.Util;
+using Semmle.Extraction.Kinds;
+
+
+namespace Semmle.Extraction.CSharp.Entities.Expressions
+{
+    internal sealed class ImplicitToString : Expression
+    {
+        /// <summary>
+        /// Gets the `ToString` method for the given type.
+        /// </summary>
+        private static IMethodSymbol? GetToStringMethod(ITypeSymbol? type)
+        {
+            return type?
+                .GetMembers()
```

Comment:
Does this include inherited members?

Reply:
No, it doesn't.

Reply:
Maybe we should consider to support this as well - should be fairly easy.

Reply:
Also good observation @hvitved !! 👍

Reply:
> Maybe we should consider to support this as well - should be fairly easy.

Yes, I think we should support that as well.

Reply:
Follow up https://github.com/github/codeql/pull/18508